### DISCUSSION TITLE
Do I have to disclose my real name in the user profile to apply for the student package?

### DISCUSSION THREAD

**Discussion by EikoocS** [community/community#60209] [GitHub Education]
### Hi everyone,

I have tried seven applications, going through the process of translating the relevant files, switching to mobile devices to submit, writing my Github username in the files, etc.
And now he prompts to ask me to make my real name in the file public in the user profile to continue the application. Is this required? Is there any other way to apply that does not require my real name to be published in my user profile. I do not want to do this.
For example, is it possible to place identification-type documents on the application file?

**Comment by sinni800** [comment]
I've hit the same snag, I really don't want to attach my real name to this account, but maybe I'll just make a new one for my real identity in the end

**Comment by Matt-Osborn** [comment]
I think you only need a valid school email address.

on the info page:

https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

it just says you have to: _"Have a verifiable school-issued email address"_

 So if your real full name is in your school email that could be an issue. But otherwise you should be good.

**Comment by opticaldrive** [comment]
Same question.

**Comment by calishu** [comment]
Same here
